src/main/java/org/sonarlint/intellij/config/project/SonarLintProjectBindPanel.java

```diff
@@ -284,6 +299,27 @@ public void actionPerformed(ActionEvent e) {
       WEST, HORIZONTAL, insets, 0, 0));
   }
 
+  private void exportConfiguration() {
+    if (project == null) return;
+
+    runOnUiThread(project, () -> {
+      if (confirm(project)) {
+        System.out.println("Confirmed to share");
+      }
+    });
+  }
+
+  private Boolean confirm(Project project) {
+    return MessageDialogBuilder.okCancel(
+        "Share this Connected Mode configuration?",
+        "A configuration file will be created in this working directory, making it easier for other team members to configure the binding" +
```

Comment:
We should add the information about the connection and project key that is going to be shared